### ISSUE TITLE
Funcionalidad

### ISSUE THREAD

**Issue by satoritechmx** [Selexur007/pokedex#1]
Muchas gracias por tu prueba, podrías complementarla con que cada 30 segundos recupere de forma automática (sin apretar un botón) al pokemón, esto igual de forma aleatoria?

**Comment by Selexur007** [comment]
Claro, entonces con un tiempo de 30 segundos vaya cambiando de Pokémon,
para cuándo lo entregaría?

**Comment by Selexur007** [comment]
https://github.com/Selexur007/pokedex

Hola que tal?, les comparto el link de la complementación que me pidieron
del pokédex
tengan una excelente tarde, les mando un cordial saludo
quedo atento a sus comentarios
